sgt-back-end/index.js

```diff
@@ -0,0 +1,142 @@
+const express = require("express")
+const app = express()
+
+const pg = require("pg")
+const db = new pg.Pool({
+  connectionString: "postgres://dev:lfz@localhost/studentGradeTable"
+})
+
+app.use(express.json())
+
+app.get("/api/grades",(req,res)=>{
+  const sql = `
+    select *
+    from "grades"
+  `
+
+  db.query(sql)
+    .then(result=>{
+      const grades = result.rows
+      res.status(200).json(grades)
+    })
+
+    .catch(err=>{
+      res.status(500).json({error: "An unexpected error occurred."})
+    })
+})
+
+
+app.post("/api/grades", (req, res) => {
+  if ("name" in req.body && "course" in req.body && "score" in req.body &&
```

Comment:
1. This is a very impressive conditional statement!
2. It should be broken into different checks, because of two things, maintainability should the code need to be refactored, and ease of debugging.